dev/tasks/jars/build-java.sh

```diff
@@ -23,12 +23,14 @@ CPP_BUILD_DIR=$GITHUB_WORKSPACE/arrow/dist/
 
 pushd java
   # build the entire project
-  mvn clean install -q -DskipTests -P arrow-jni -Darrow.cpp.build.dir=$CPP_BUILD_DIR
-  # test only gandiva
-  mvn test -q -P arrow-jni -pl gandiva -Dgandiva.cpp.build.dir=$CPP_BUILD_DIR
+  mvn clean install -DskipTests -P arrow-jni -Darrow.cpp.build.dir=$CPP_BUILD_DIR
```

Comment:
I think we can move this file to the `ci/scripts` directory as well using the same prefix as the cpp build scripts.

Reply:
The script was moved to the folder